scripts/db-clone/clone-prod-to-staging.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# This script orchestrates the cloning of Keycloak and Loculus databases from production to staging
+# Keycloak is dumped second and loaded first to prevent potential race conditions
+
+# Set variables
+CHILD_SCRIPT="./clone.sh"
+PROD_KC_DUMP="production_keycloak_dump.sql"
+PROD_LOC_DUMP="production_loculus_dump.sql"
+PROD_KC_DB="pathoplexus_prod_keycloak"
+PROD_LOC_DB="pathoplexus_prod_loculus"
+STAGING_KC_DB="pathoplexus_staging_keycloak"
+STAGING_LOC_DB="pathoplexus_staging_loculus"
+STAGING_KC_USER="staging_keycloak_user"
+STAGING_LOC_USER="staging_loculus_user"
+
+check_command() {
+    if [ $? -ne 0 ]; then
+        echo "Error: $1"
+        exit 1
+    fi
+}
+
+# Note: Could screw up columns and values that contain `prod` etc
+# For now not an issue but might eventually want to be more surgical
```

Comment:
Is this not critical and can it not potentially be used for an attack? If someone, for example, has "prod" in their email (let's say `prodigy@gmail.com`) and this get's replaced to `stagingidy@gmail.com`, someone can potentially create the `stagingidy@gmail.com` address to reset the password and get access to the account. In combination with https://github.com/pathoplexus/private_issues/issues/7#issuecomment-2371978447, this can potentially even allow access to the account on the production server.

Reply:
Could we instead first load the new database and then use SQL for a few targeted replacements?